test/InterchainTokenService.js

```diff
@@ -2674,6 +2674,29 @@ describe('Interchain Token Service', () => {
             expect(await tokenManager.hasRole(otherWallet.address, FLOW_LIMITER_ROLE)).to.equal(true);
         });
 
+        it('Should be able to transfer a flow limiter', async () => {
+            await expect(tokenManager.transferFlowLimiter(otherWallet.address, wallet.address))
+                .to.emit(tokenManager, 'RolesAdded')
+                .withArgs(wallet.address, 1 << FLOW_LIMITER_ROLE)
+                .to.emit(tokenManager, 'RolesRemoved')
+                .withArgs(otherWallet.address, 1 << FLOW_LIMITER_ROLE);
+
+            expect(await tokenManager.hasRole(wallet.address, FLOW_LIMITER_ROLE)).to.equal(true);
+            expect(await tokenManager.hasRole(otherWallet.address, FLOW_LIMITER_ROLE)).to.equal(false);
+        });
+
+        it('Should revert when trying to transfer from an address that is not the flow limiter', async () => {
+            await expectRevert(
+                (gasOptions) => tokenManager.transferFlowLimiter(otherWallet.address, wallet.address, gasOptions),
+                tokenManager,
+                'MissingAllRoles',
+                [otherWallet.address, 1 << FLOW_LIMITER_ROLE],
+            );
+
+            expect(await tokenManager.hasRole(wallet.address, FLOW_LIMITER_ROLE)).to.equal(true);
+            expect(await tokenManager.hasRole(otherWallet.address, FLOW_LIMITER_ROLE)).to.equal(false);
+        });
+
```

Comment:
add a multicall test, such as transferring flow limiter and operator in one call?

Reply:
added